mesonbuild/interpreter/interpreter.py

```diff
@@ -674,19 +674,12 @@ def extract_variables(self, kwargs: T.Dict[str, T.Union[T.Dict[str, str], T.List
     )
     def func_declare_dependency(self, node: mparser.BaseNode, args: T.List[TYPE_var],
                                 kwargs: kwargs.FuncDeclareDependency) -> dependencies.Dependency:
-        deps = kwargs['dependencies']
         incs = self.extract_incdirs(kwargs)
-        libs = kwargs['link_with']
-        libs_whole = kwargs['link_whole']
-        sources = self.source_strings_to_files(kwargs['sources'])
-        compile_args = kwargs['compile_args']
-        link_args = kwargs['link_args']
         variables = kwargs['variables']
         version = kwargs['version']
+        sources = self.source_strings_to_files(kwargs['sources'])
```

Comment:
Is there a reason this gets deleted from one hunk and added lower down in a different hunk?